charts/umbrella/README.md

```diff
@@ -330,7 +338,28 @@ TBD.
 
 #### Get to know the Portal
 
-Perform first login and send out an invitation to a company to join the network (SMTP account required to be configured in custom values.yaml file).
+Perform first login and send out an invitation to a company to join the network (SMTP account required to be configured in custom values.yaml file). Since the onboarding process requires the clearinghouse to work properly, but clearinghouse currently isn't available as a FOSS application you can skip the step with the following SQL Script which must be executed against the portal database.
```

Comment:
could you please add the whole information after the "Proceed with the login..." part?
In a section that might called something like "Note for onboarding process"

Clearinghouse should also b camelcase and the link [here](https://github.com/eclipse-tractusx/portal-assets/blob/v2.1.0/docs/developer/Technical%20Documentation/Interface%20Contracts/Clearinghouse.md) might also be useful.

Reply:
adjusted